### PR TITLE
Bug fix for sending the same correlation id in historian client

### PR DESCRIPTION
**Pull request by PingZhu2232**
In historian client now, default headers include a fixed correlation id during construction. This fix is to pass getCorrelationId to RestWrapper and calls it when sending requests to get correlation id in the current thread (function chain).

### CODE REVIEW

File: server/routerlicious/packages/services/src/tenant.ts [hunk 1]
```diff
@@ -63,12 +62,18 @@ export class TenantManager implements core.ITenantManager {
             token: fromUtf8ToBase64(`${credentials.user}`),
         };
         const defaultHeaders = {
-            "Authorization": getAuthorizationTokenFromCredentials(credentials),
-            // TODO: all requests from this GitManager will have the same correlation id. Is this intended behavior?
-            "x-correlation-id": getCorrelationId() || uuid(),
+            Authorization: getAuthorizationTokenFromCredentials(credentials),
```
Comment (hedasilv) [inline server/routerlicious/packages/services/src/tenant.ts:65]: Curious, any reason to remove the quotes here?

**Review by hedasilv — APPROVED** [review]

File: server/routerlicious/packages/services/src/tenant.ts (hunk 1 above)
Comment (PingZhu2232) [reply server/routerlicious/packages/services/src/tenant.ts:65]: It didn't pass eslint... showed "Properties shouldn't be quoted as all quotes are redundant", but it should be the same without quotes.

**Review by tanviraumi — APPROVED** [review]
Looks good. @znewton Feel free to merge if looks good to you

**Review by pradeepvairamani — APPROVED** [review]

File: server/routerlicious/packages/services/src/tenant.ts (hunk 1 above)
Comment (hedasilv) [reply server/routerlicious/packages/services/src/tenant.ts:65]: Got it. Makes sense!

**Review by znewton — APPROVED** [review]

**Pull request merged**